doc/workshops/tutorial/content/intro/multi_fluid.md

```diff
@@ -0,0 +1,397 @@
+
+!---
+
+# Kernels
+
+!---
+
+# Drift-Diffusion Approximation
+
+# style=margin-top:2em;
+
+\begin{equation}
+  \pf{n_i}{t}
+  +
+  \nabla \cdot \Gamma_i
+  =
+  \sum_j
+  S_{j,n}
+\end{equation}
+
+$i$ and $j$ denote some particle species, $n$ denotes the number of bodies in the source term, and $S_{j}$ denotes sources of species $j$ from chemical reactions.
+
+\begin{equation}
+  \Gamma_{\alpha} =
+  \mu_{\alpha}
+  \nabla \phi\;
+  n_\alpha
+  -
+  D_\alpha
+  \nabla n_\alpha
+\end{equation}
+
+$\mu_\alpha$ and $D_\alpha$ are the mobility and diffusivity of species $\alpha$ respectively, and $\phi$ is the electric potential.
+
+\begin{equation}
+  \Gamma_\beta =
+  -
+  D_\beta
+  \nabla n_\beta
+\end{equation}
+
+$\alpha$ denotes charged particles, beta denotes neutral particles.
+
+!---
+
+# Source Terms (Crane)
+
+# style=margin-top:1em;
+
+\begin{equation}
+  S_{i,1}
+  =
+  \nu
+  k
+  n_1
+\end{equation}
+
+\begin{equation}
+  S_{i, 2}
+  =
+  \nu
+  k
+  n_1 n_2
+\end{equation}
+
+\begin{equation}
+  S_{i, 3}
+  =
+  \nu
+  k
+  n_1 n_2 n_3
+\end{equation}
+
+Where $\nu$ is the stoichiometric coefficient of the reaction, $k$ is the reaction rate, and $n_i$ is the density of species $i$
+
+!---
+
+# Electro-Magnetic Field Calculations
+
+When evaluating an electrostatic system Poisson's equation is used to calculate the electric potential.
+
+\begin{equation}
+  - \nabla^2 \phi
+  =
+  \frac{\rho}{\varepsilon_0}
+\end{equation}
+
+Where $e$ is the elementary charge, $\varepsilon_0$ is the permitivity of free space, and $\rho$ is the charge density in the system.
+
+Additionally, Zapdos is capable of calculating an effective electric field using the form
+
+\begin{equation}
+  \pf{\vec{E}_\text{eff}}{t}
+  =
+  \nu_\text{ion}
+  \left(
+    \vec{E} - \vec{E}_\text{eff}
+  \right)
+\end{equation}
+
+
+In cases where electromagnetic systems are considered MOOSE's [Electromagnetics Module](https://mooseframework.inl.gov/modules/electromagnetics/) is used to perform field calculations
+
+!---
+
+# Energy Conservation
+
+# style=margin-top:2em;
+
+\begin{equation}
+  \pf{n_e \varepsilon}{t}
+  +
+  \nabla \cdot \Gamma_\varepsilon
+  =
+  - e \Gamma_e
+  \cdot
+  \nabla
+  \phi
+  -
+  3
+  \frac{m_e}{m_i}
+  n_e n_g
+  k_\text{elastic}
+  T_e
+  -
+  \sum_i  E_i K_i
+\end{equation}
+
+\begin{equation}
+  \Gamma_\epsilon
+  =
+  \frac{5}{3}
+  \left(
+  \epsilon
+  \Gamma_e
+  -
+  n_e
+  D_e
+  \nabla n_e
+  \right)
+\end{equation}
+
+!---
+
+# Boundary Conditions (BCs)
+
+!---
+
+# Electrostatic Potential Boundary Conditions
+
+# style=margin-top:2em;
+
+\begin{equation}
+  \phi =
+  \phi_0
+  \sin
+  \left(
+    2 \pi
+    f t
+  \right)
+\end{equation}
+
+\begin{equation}
+  \phi = 0
+\end{equation}
+
+\begin{equation}
+  e \Gamma_i
+  -
+  e \Gamma_e
+  +
+  \varepsilon_0
+  \pf{\left(\nabla \phi\right)}{t}
+  =
+  \frac{\varepsilon_i}{d_i}
+  \pf{\phi}{t}
+\end{equation}
+
+!---
+
+# Flux Boundary Conditions: Type 1
```

Comment:
Commenting again because I used this terminology in the original PowePoint Slides. For context, the original slides only briefly show the multi-fluid equation and BC equations as two slide with just equations. The idea being that these students just had an hour and half lecture on the fluid approach to plasma modeling and that this just shows that we currently have 3 sets of difference boundary condition assumptions within Zapdos.

@gsgall While #250 is still under review, the descriptions of these BCs are should be done, so you could 'copy-paste' or summaries those descriptions here.

Reply:
Let's resolve this for now and add it after #250 is in.